roadmap.md

```diff
@@ -7,106 +7,32 @@ In November 2017, WebAssembly CG members representing four browsers, Chrome, Edg
-        }
-      }
-
-      return { matched, unmatched };
-    }
-
-    let { features, browsers } = await fetch('/features.json').then(res => res.json());
-    let table = document.getElementById('feature-support');
-
-    function h(name, props = {}, children = []) {
-      let node = Object.assign(document.createElement(name), props);
-      node.append(...children);
-      return node;
-    }
-
-    let tBody = document.createElement('tbody');
-
-    table.append(
-      h('thead', {}, [
-        h('tr', {}, [
-          h('th'),
-          h('th', {}, ['Your browser']),
-          ...Object.entries(browsers).map(([name, { url, logo, version }]) =>
-            h('th', {}, [
-              h('a', { href: url }, [
-                h('img', { src: logo, width: 32, height: 32 }),
-                h('br'),
-                name,
-                h('sup', {}, [version]),
-              ])
-            ])
-          )
-        ])
-      ]),
-      tBody
-    );
-
-    let featureGroups = partitionArray(
-      Object.entries(features).map(([name, feature]) => Object.assign(feature, { name })),
-      feature => feature.phase >= 4
-    );
-
-    featureGroups = [
-      { name: 'Standardized features', features: featureGroups.matched },
-      { name: 'In-progress proposals', features: featureGroups.unmatched },
-    ];
-
-    const columnCount = 2 + Object.keys(browsers).length;
-
-    for (let { name, features } of featureGroups) {
-      tBody.append(
-        h('tr', {}, [
-          h('th', { colSpan: columnCount }, [name])
-        ])
-      );
-      for (let { name, description, url, phase } of features) {
-        let supportHTML = h('td');
-        Promise.resolve()
-          // Make sure to call `wasmFeatureDetect` inside a promise-chained
-          // function so that we can still render table rows for features that
-          // we don't have a detector yet.
-          .then(() => wasmFeatureDetect[name]())
-          .then(
-            supported => (supported ? '✔️' : '❌'),
-            err => '❌'
-          )
-          .then(textContent => {
-            supportHTML.textContent = textContent;
-          });
-        tBody.append(
-          h('tr', {}, [
-            h('th', {}, [h('a', { href: url }, [description])]),
-            supportHTML,
-            ...Object.values(browsers).map(({ features }) => {
-              let support = features[name];
-              if (typeof support === 'string') {
-                return h('td', { title: support, tabIndex: 0 }, ['⏳']);
-              }
-              return h('td', {}, [support ? '✔️' : support === null ? 'ⁿ/ₐ' : '❌']);
-            })
-          ])
-        );
-      }
-    }
-  })();
-</script>
+<!-- Apache License 2.0, https://github.com/Remix-Design/remixicon -->
+<template id="support-symbol-yes">
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-label="Supported"><path class="svg-stroke" d="M10 15.2 19.2 6l1.4 1.4L10 18l-6.4-6.4L5 10.2Z"/></svg>
```

Comment:
When I navigate to cells that just contain `✓` or `✗`, I hear e.g. "supported, blank cell" in Narrator. I think it's because those `aria-label`s live on those SVG icons, but not on the cells themselves. Worth trying to move them over?

Reply:
Ah, I tested it with NVDA instead of Narrator, and it didn't have that issue. I found the exact wording can differ a lot depending on the screen reader/browser/operating system, so it's really hard to find a combination that works perfectly in all situations.